lib/teaspoon/formatter/base.rb

```diff
@@ -154,6 +154,25 @@ def filename(file)
         filename += "?#{params.join("&")}" if params.any?
         filename
       end
+
+      def parse_output_file(output)
+        return output unless output
+        match = output.scan(/%{([^}]*)}/)
```

Comment:
Any reason to use scan instead of gsub with a proc?

```
output.gsub!(/%{([^}]*)}/) { |cap| parse_output_capture(cap) }
```

Then you can extract the switch (aka case) statement into a separate method because I'm not a fan of the indentation style. =)